numpy/random/src/distributions/distributions.c

```diff
@@ -342,31 +342,32 @@ uint64_t random_uint(bitgen_t *bitgen_state) {
  * using logfactorial(k) instead.
  */
 double random_loggam(double x) {
-  double x0, x2, xp, gl, gl0;
+  double x0, x2, xlp, gl, gl0;
   RAND_INT_TYPE k, n;
 
   static double a[10] = {8.333333333333333e-02, -2.777777777777778e-03,
                          7.936507936507937e-04, -5.952380952380952e-04,
                          8.417508417508418e-04, -1.917526917526918e-03,
                          6.410256410256410e-03, -2.955065359477124e-02,
                          1.796443723688307e-01, -1.39243221690590e+00};
-  x0 = x;
-  n = 0;
+
   if ((x == 1.0) || (x == 2.0)) {
     return 0.0;
-  } else if (x <= 7.0) {
+  } else if (x < 7.0) {
     n = (RAND_INT_TYPE)(7 - x);
-    x0 = x + n;
+  } else {
+    n = 0;
   }
-  x2 = 1.0 / (x0 * x0);
-  xp = 2 * M_PI;
+  x0 = x + n;
+  x2 = (1.0 / x0) * (1.0 / x0);
+  xlp = 1.8378770664093453e+00;
```

Comment:
A comment may be in order; 2\*M_PI is very easy to read. Might not guess this is log(2*pi)

Reply:
Yes I was wondering if adding a comment or not, but since there wasn't any around I choose not to, but I totally agree

Reply:
don't care too much, but naming it `log_2pi` would make the formula below probably read better than before, rather than worse.

Reply:
Hmm, unless `p` has a meaning in this context, so maybe nevermind...